docs/guides/developer/docs/reviewing-and-merging.md

```diff
@@ -34,8 +34,7 @@ Reviewing Rules
 Reviewing a Pull Request
 ------------------------
 
-There is no list of things you are required to do as reviewer of a pull request. Our primary goals in doing reviews for
```

Comment:
Streamline: "There is no list of things you are required to do as reviewer of a pull request." really isn't helpful.

Reply:
The intent behind this sentence was to say that reviewers should first and foremost think about the change and assess if the change makes sense and will not harm (short or long-term) anyone which is not just checking a list. Additional, it also means that as a reviewer, you can be pragmatic and, if it makes sense in that situation, you can ignore all other listed items here. Being pragmatic is sometimes better than just sticking to rules.

Reply:
Just undone changes that are not directly relevant to the proposal